packages/react-renderer-demo/src/next.config.js

```diff
@@ -69,7 +69,6 @@ module.exports = withBundleAnalyzer(
 
         config.optimization.minimizer = [
           new TerserPlugin({
-            cache: true,
```

Comment:
@Hyperkid123 was this important? 😄 The option does not exist anymore and there is no migration guide to it.

Reply:
NO, it was not. There is the persistent environment (it's not configured) so there is nothing to cache really.